src/app/pages/overlays/complex-bottom-sheet/bottom-sheet/bottom-sheet.html

```diff
@@ -1,65 +1,60 @@
 <mat-nav-list>
     <div class="footer-list" data-cy="btm-sheet-sort">
-        <h3 class="mat-overline">Sort by</h3>
-        <div fxLayout="row wrap" fxLayoutAlign="center center">
-            <a [class.active]="activeSort == filterTypes.TIME" (click)="sortData(filterTypes?.TIME)">
-                <button mat-icon-button>
-                    <mat-icon mat-list-icon>access_time</mat-icon>
+        <div class="mat-h3">Sort By:</div>
+        <pxb-hero-banner>
+            <pxb-hero label="Time" [iconSize]="36" [class.active]="activeSort == filterTypes.TIME">
+                <button pxb-primary mat-icon-button (click)="sortData(filterTypes?.TIME)">
+                    <mat-icon mat-list-icon [style.fontSize.px]="24">access_time</mat-icon>
                 </button>
-                <span mat-line style="display: block;">Time</span>
-            </a>
-            <a [class.active]="activeSort == filterTypes.EVENT_TYPE" (click)="sortData(filterTypes.EVENT_TYPE)">
-                <button mat-icon-button>
-                    <mat-icon data-cy="btm-sheet-type-btn" mat-list-icon>info</mat-icon>
+            </pxb-hero>
```

Comment:
The whole hero should be clickable, rather than just the icon
- You'll probably want to convert those mat-icon-buttons to plain mat-icons when you make this change

Reply:
I've made the entire Hero clickable; no more mat-buttons.